File: Teledyne DALSA/Sapera/Demos/NET/BayerDemo/CSharp/BayerDemo.cs
using System;
using System.Collections.Generic;
using System.Windows.Forms;

namespace DALSA.SaperaLT.Demos.NET.CSharp.BayerDemo
{
    static class BayerDemo
    {
        /// <summary>
        /// The main entry point for the application.
        /// </summary>
        [STAThread]
        static void Main()
        {
            Application.EnableVisualStyles();
            Application.SetCompatibleTextRenderingDefault(false);
            BayerDemoDlg dlg = new BayerDemoDlg();
            if(!dlg.IsDisposed)
                Application.Run(dlg);
        }
    }
}

File: Teledyne DALSA/Sapera/Demos/NET/BayerDemo/VB/BayerDemo.vb
Public Class BayerDemo
   'The main entry point for the application.
   <STAThread()> _
   Shared Sub Main()
      Application.EnableVisualStyles()
      Application.SetCompatibleTextRenderingDefault(False)
      Dim BD As BayerDemoDlg = New BayerDemoDlg()
      If Not BD.IsDisposed Then
         Application.Run(BD)
      End If
   End Sub
End Class
